src/lib/core/overlay/position/connected-position-strategy.ts

```diff
@@ -89,11 +95,18 @@ export class ConnectedPositionStrategy implements PositionStrategy {
   }
 
   /** Sets the layout direction so the overlay's position can be adjusted to match. */
-  setDirection(dir: 'ltr' | 'rtl') {
+  withDirection(dir: 'ltr' | 'rtl'): ConnectedPositionStrategy {
     this._dir = dir;
     return this;
   }
 
+  /** Sets an offset for the overlay's connection point */
+  withOffset(offset: Point): ConnectedPositionStrategy {
+    this._offsetX = offset.x;
+    this._offsetY = offset.y;
+    return this;
+  }
```

Comment:
Thinking about it more, what do you think of having separate methods for x and y? As it is now, you have to specify both of them.